examples/oft-solana/tasks/solana/createOFT.ts

```diff
@@ -101,6 +114,13 @@ task('lz:oft:solana:create', 'Mints new SPL Token and creates new OFT Store acco
     .addParam('sellerFeeBasisPoints', 'Seller fee basis points', 0, devtoolsTypes.int)
     .addParam('symbol', 'Token Symbol', 'MOFT', devtoolsTypes.string)
     .addParam('tokenMetadataIsMutable', 'Token metadata is mutable', true, devtoolsTypes.boolean)
+    .addParam('additionalMinters', 'Comma-separated list of additional minters', '', devtoolsTypes.string)
```

Comment:
I think we have a devtoolsTypes.csv type

Reply:
Fixed here and in `setAuthority.ts` for symmetry.